app/invocation/invocation_details_card.tsx

```diff
@@ -33,17 +34,40 @@ export default class ArtifactsCardComponent extends React.Component<Props, State
     alert_service.success("Command line copied to clipboard!");
   }
 
-  bazelCommandAndPatternWithOptions(options: string[]) {
-    return [
-      "bazel",
-      this.props.model.started?.command,
-      ...(this.props.model.expanded?.id?.pattern?.pattern || []),
-      ...(options || []),
-    ]
-      .filter((value) => value)
+  // Wraps arguments containing spaces in the provided command-line in
+  // quotation marks so they work when copied and pasted. The input
+  // command-line is passed in as an array with one entry per piece. For
+  // example, this command:
+  //   "bazel build --output_filter='argument with spaces' //..."
+  // is passed into this function as:
+  //   ["bazel", "build", "--output_filter=argument with spaces"," "//..."],
+  // and this will be returned:
+  //   ["bazel", "build", "--output_filter='argument with spaces'"," "//..."],
+  quote(pieces: string[]) {
+    return pieces
+      .map((value) => {
+        if (value.includes("=") && value.includes(" ")) {
```

Comment:
nit: remove ` && value.includes(" ")` so that other chars like `"` or `'` get quoted too (if the part after the `=` doesn't need quoting then shlex will leave it untouched)

Reply:
Ah good call, done.